db/migrate/20221122151644_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] The references should point to an `author`, not `user`. You can use the `to_table` option to point the database to the User table.

Reply:
Thank you for the review @DammyShittu